### ISSUE TITLE
NTR: Urine osmolality inappropriately low after vasopressin (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8832]
New term request Urine osmolality inappropriately low after vasopressin (elthat_100530015828): DDAVP®; desmopressin; antidiuretic hormone (ADH); arginine vasopressin

**Comment by pnrobinson** [comment]
@MickeySegal
This is a near synonym of [Nephrogenic diabetes insipidus HP:0009806](https://hpo.jax.org/app/browse/term/HP:0009806). Possibly it is sufficient to have Inappropriately low urine osmolality following vasopressin test ?

**Comment by MickeySegal** [comment]
Nephrogenic diabetes insipidus is more like a diagnosis, while Inappropriately low urine osmolality following vasopressin test is a lab finding.

**Comment by pnrobinson** [comment]
@MickeySegal 
slightly modified for generality

Adding new term: 
- ID: HP:6001079
- label: Inappropriately low urine osmolality following challenge
- parents: [Abnormal urine osmolality (HP:0033358)](https://hpo.jax.org/app/browse/term/HP:0033358)
- pmid: PMID:16461963
- def: In diabetes insipidus, affected individuals produce large quantities of dilute urine. In nephrogenic DI, the kidneys do not respond to vasopressin. In a water deprivation test or with administration of exogenous vasopressin, the osmolaltiy of the urine does not increase as normally.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8832
- synonyms:
